airflow/dags/create_external_tables/elavon_transactions.yml

```diff
@@ -0,0 +1,105 @@
+operator: operators.ExternalTable
+bucket: "{{ env_var('CALITP_BUCKET__ELAVON') }}"
+suffix_bucket: true
+source_objects:
+  - "transactions/*.jsonl.gz"
+destination_project_dataset_table: "external_elavon.transactions"
+source_format: NEWLINE_DELIMITED_JSON
+use_bq_client: true
+hive_options:
+  mode: CUSTOM
+  source_uri_prefix: "transactions/{dt:DATE}/{execution_ts:TIMESTAMP}/"
+schema_fields:
+  - name: payment_reference
+    type: STRING
+  - name: payment_date
+    type: STRING
+  - name: account_number
+    type: STRING
+  - name: routing_number
+    type: STRING
+  - name: fund_amt
+    type: STRING
+  - name: batch_reference
+    type: STRING
+  - name: batch_type
+    type: STRING
+  - name: customer_batch_reference
+    type: STRING
+  - name: customer_name
+    type: STRING
+  - name: merchant_number
+    type: STRING
+  - name: external_mid
+    type: INTEGER
```

Comment:
on littlepay I hit issues with incorrect files so I typed everything as STRING and safe_cast in staging models; that might be useful here

Reply:
Addressed in [a21ab0f](https://github.com/cal-itp/data-infra/pull/2376/commits/a21ab0f9815f91d81721ab26c85fd426f54010b2)